jibri/Dockerfile

```diff
@@ -15,12 +15,37 @@ ARG USE_CHROMIUM=0
 ARG CHROME_RELEASE=114.0.5735.90
 ARG CHROMEDRIVER_MAJOR_RELEASE=114
 
-COPY rootfs/ /
+COPY --chown=noroot:noroot rootfs/ /
 
 RUN apt-dpkg-wrap apt-get update && \
     apt-dpkg-wrap apt-get install -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" jibri libgl1-mesa-dri procps jitsi-upload-integrations jitsi-autoscaler-sidecar jq pulseaudio dbus dbus-x11 rtkit unzip fonts-noto && \
     /usr/bin/install-chrome.sh && \
-    apt-cleanup && \
-    adduser jibri rtkit
+    apt-cleanup
+
+RUN mkdir -p /etc/jitsi/autoscaler-sidecar
+RUN mkdir -p /config/logs
+RUN mkdir -p /config/recordings
+RUN mkdir -p /jibri
+
+RUN chown -R noroot:noroot /etc/cont-init.d
```

Comment:
Many of these look generic enough to do it in the base image, any reason not to do it there?